python/tvm/driver/tvmc/workspace_pools.py

```diff
@@ -0,0 +1,199 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Functions for processing dynamic workspace pool TVMC args
+"""
+
+
+import logging
+import re
+
+from tvm.driver.tvmc import TVMCException
+from tvm.ir.memory_pools import PoolInfo, WorkspaceMemoryPools
+
+
+# pylint: disable=invalid-name
+logger = logging.getLogger("TVMC")
+
+
+def generate_workspace_pools_args(parser):
+    """Generates arguments for each Workspace Pools's options"""
+    parser.add_argument(
```

Comment:
is it possible to specify these in a file?

Reply:
I'm a little unsure by what you mean here, could you please elaborate? This approach using `parser.add_argument()` is taken from how other command line arguments are captured

Reply:
ah yeah, maybe my question was bigger than this PR. just was curious if it would be possible to configure via e.g. yaml or json here. likely it would be a target-wide file, so perhaps we should table this question and come back to it in the future.